cmd/gen/makefile/template/template.go

```diff
@@ -0,0 +1,67 @@
+package template
```

Comment:
What do you folks think about running `go mod tidy` automagically after building?